pkg/templates/aws/resources/api_integration.yaml

```diff
@@ -24,16 +25,20 @@ properties:
             properties:
               FullPath: '{{ fieldValue "Route" .Self }}'
               RestApi: '{{ fieldValue "RestApi" .Self }}'
+    description: The identifier of an API resource. This is usually set when the integration
+      needs to reference a specific resource within an API
   Method:
     type: resource(aws:api_method)
     operational_rule:
       step:
         direction: upstream
         resources:
           - selector: aws:api_method
-            properties: 
+            properties:
               RestApi: '{{ fieldValue "RestApi" .Self }}'
-        unique: true # Each Integration must have its own method
```

Comment:
this got removed unintenionally i assume, all the other removals look like just line changes